.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: slow-tests
+on: [workflow_dispatch]
+jobs:
+  test-illumos:
+    runs-on: macos-latest
+    name: Test in illumos VM
+    steps:
+    - uses: actions/checkout@v2
+    - name: Test on illumos
+      id: test
+      uses: papertigers/illumos-vm@main
+      with:
+        prepare: |
+          pkg set-publisher -r -O https://pkg.omnios.org/r151036/extra extra.omnios
```

Comment:
@papertigers Do you include the extra repo in the latest images so that I could skip this part?
Alternately is there a clever way to make this more generic to survive across releases?

Reply:
Following up, @papertigers was kind enough to do this for the @r38 branch using a newer OmniOS release:
https://github.com/papertigers/fsnotify/commit/2db24b5fb78a888b94c923fa9e6e9b08d8abc554

I will pull in those changes soon.